roles/pulibrary.tomcat8/tasks/main.yml

```diff
@@ -1,5 +1,4 @@
 ---
-# tasks file for roles/pulibrary.tomcat8
 - name: ensure tomcat8 packages are installed
```

Comment:
https://wiki.duraspace.org/display/DSDOC6x/Installing+DSpace#InstallingDSpace-ServletEngine(ApacheTomcat7orlater,Jetty,CauchoResinorequivalent) doesn't list any information referencing Tomcat 9, but I would be curious about testing this against DSpace 6.x or 7.x releases.